Objects/unionobject.c

```diff
@@ -137,102 +137,155 @@ union_richcompare(PyObject *a, PyObject *b, int op)
     return result;
 }
 
-static PyObject*
-flatten_args(PyObject* args)
+static int
+is_same(PyObject* left, PyObject* right) {
```

Comment:
PEP 7: Function definition style: function name in column 1, outermost curly braces in column 1, blank line after local variable declarations.

```suggestion
is_same(PyObject* left, PyObject* right)
{
```